hips/draw/simple.py

```diff
@@ -1,5 +1,122 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
 
+import numpy as np
+from skimage import transform as tf
+from typing import List, Generator, Any
+
+from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
+from ..utils import WCSGeometry, compute_healpix_pixel_indices, frames
+
 __all__ = [
+    'draw_sky_image',
+    'make_sky_image',
+    'SimpleTilePainter'
 ]
+
+
+def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any]) -> np.ndarray:
+    """Draw sky image using the simple and quick method.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tiles : List[HipsTile]
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    # TODO: Fix type annotation issue
+    sky_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        painter = SimpleTilePainter(geometry, tile)
+        sky_sky += painter.warp_image()
+    return sky_sky
+
+
+class SimpleTilePainter:
+    """Paint a single tile using a simple projective transformation method.
+
+    The algorithm implemented is described here: :ref:`drawing_algo`.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tile : `HipsTile`
+       An object of HipsTile
+    """
+
+    def __init__(self, geometry: WCSGeometry, tile: HipsTile) -> None:
+        self.geometry = geometry
+        self.tile = tile
+        self.pt = None
```

Comment:
Remove `self.pt` here, and also in the property below, simply use a local variable `pt`, don't store it on `self`.